src/mumble/Log_macx.mm

```diff
@@ -71,7 +71,12 @@ static bool growl_available() {
 void Log::postNotification(MsgType mt, const QString &plain) {
 	QString title = msgName(mt);
 #if MAC_OS_X_VERSION_MAX_ALLOWED >= 1080
+# if QT_VERSION >= QT_VERSION_CHECK(5, 9, 0)
+	const QOperatingSystemVersion current = QOperatingSystemVersion::current();
+	if (!(current < QOperatingSystemVersion::OSXMavericks)) {
```

Comment:
This is not equivalent to the old comparison, Mountain Lion is not listed in `QOperatingSystemVersion`.

We should probably use `if (current.majorVersion() >= 10 && current.minorVersion() >= 8)`.